android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.java

```diff
@@ -70,6 +116,13 @@ protected void onStop() {
         super.onStop();
     }
 
+    @Override
+    public void onAttachedToWindow() {
+        super.onAttachedToWindow();
+
+        mKeepScreenOnHandler.post(mKeepScreenOnUpdater);
```

Comment:
Just tested now, if you put preventsDisplaySleepDuringVideoPlayback={false} and go to fullscreen, preventsDisplaySleepDuringVideoPlayback is not correctly applied

Reply:
I can propose your this aditionnal patch patch: https://github.com/react-native-video/react-native-video/pull/3596/commits/bceb070aa34d47ee07351330264b3b0a73ff775b

I can merge my PR if you want to avoid cherry-picking ! :)

Reply:
Good catch, I was not aware of `preventsDisplaySleepDuringVideoPlayback` props.